backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java

```diff
@@ -39,7 +39,7 @@
       AttributeType.TOXICITY, 
       AttributeType.UNSUBSTANTIAL
     };
-
+    
```

Comment:
Accidental space added. Should not affect anything about project. Please ignore.

Reply:
It didn't remove this, but I'll go do it manually.

Reply:
Done